qa/TL3_RN50_convergence/test_tensorflow.sh

```diff
@@ -7,8 +7,9 @@ mkdir -p idx-files/
 NUM_GPUS=$(nvidia-smi -L | wc -l)
 
 DATA_SET_DIR=/data/imagenet/train-val-tfrecord
-for file in $(ls $DATA_SET_DIR --ignore *\.txt);
+for file in $(ls $DATA_SET_DIR/*-of-*);
```

Comment:
how about matching for the extension as well?

Reply:
File names are like `validation-00049-of-00128` so there is not extension we could use for the pattern.